packages/schema/bind/src/__tests__/cases/sanity/output/wasm-as/Query/wrapped.ts

```diff
@@ -1,9 +1,12 @@
 import {
-  queryMethod
+  queryMethod,
+  objectMethod
```

Comment:
This doesn't make sense currently and are unresolved imports
Will look through diff to find what was changed